pkg/v1/tkg/web/src/app/views/landing/vsphere-wizard/provider-step/vsphere-provider-step.component.ts

```diff
@@ -248,47 +252,58 @@ export class VSphereProviderStepComponent extends StepFormDirective implements O
         this.loadingState = ClrLoadingState.LOADING;
         this.vsphereHost = this.formGroup.controls['vcenterAddress'].value;
 
+        if (this.formGroup.controls['insecure'].value === false) {
```

Comment:
nit: I would invert the logic here so it's just:
`if (this.formGroup.controls['insecure'].value) { this.login() }`
because I think the code reads more easily that way